internal/controlplane/handlers_authz.go

```diff
@@ -80,13 +80,8 @@ func ProjectAuthorizationInterceptor(ctx context.Context, req interface{}, info
 
 	relation := opts.GetRelation()
 
-	relationValue := relation.Descriptor().Values().ByNumber(relation.Number())
-	if relationValue == nil {
-		return nil, status.Errorf(codes.Internal, "error reading relation value %v", relation)
-	}
-	extension := proto.GetExtension(relationValue.Options(), minder.E_Name)
-	relationName, ok := extension.(string)
-	if !ok {
+	relationName := relationAsName(relation)
+	if relationName == "" {
```

Comment:
This seems unrelated. Should it be in a different PR?

Reply:
I wanted to re-use this logic in `handlers_projects.go` to get the OpenFGA relation for a specific relation (`create`), so I extracted this bit of code.

Reply:
gotcha